dbt_subprojects/dex/models/aggregator_trades/dex_aggregator_trades.sql

```diff
@@ -44,8 +44,8 @@ SELECT
     , token_pair
     , token_bought_amount
     , token_sold_amount
-    , token_bought_amount_raw
-    , token_sold_amount_raw
+    , try_cast(token_bought_amount_raw as int256) as token_bought_amount_raw
+    , try_cast(token_sold_amount_raw as int256) as token_sold_amount_raw
```

Comment:
@jeff-dude raw_amount issue is resolved with a try_cast in `dex_aggregator_trades`. Are you ok with this approach?